### PR TITLE
day2 -all 41 exercises

### PR DESCRIPTION
**Pull request by arojecka**

### CODE REVIEW

File: 02_Day_Data_types/02_day_starter/main.js [hunk 1]
```diff
@@ -1 +1,150 @@
...
+    22.Use endsWith() method with the string 30 Days Of JavaScript and make the result true
+    23.Use match() method to find all the a’s in 30 Days Of JavaScript
+    24.Use concat() and merge '30 Days of' and 'JavaScript' to a single string, '30 Days Of JavaScript'
+    25.Use repeat() method to print 30 Days Of JavaScript 2 times
+*/
+
+let challenge = '30 Days Of JavaScript'
+console.log(challenge)
+console.log(challenge.length)
+challenge.toLowerCase
```
Comment (kRojecki) [inline 02_Day_Data_types/02_day_starter/main.js]:
```suggestion
challenge.toLowerCase()
```

File: 02_Day_Data_types/02_day_starter/main.js [hunk 2]
```diff
@@ -1 +1,150 @@
...
+challenge.toUpperCase
```
Comment (kRojecki) [inline 02_Day_Data_types/02_day_starter/main.js]:
```suggestion
challenge.toUpperCase()
```

File: 02_Day_Data_types/02_day_starter/main.js [hunk 3]
```diff
@@ -1 +1,150 @@
...
+challenge.substring(0,2)
+challenge.substring(3)
+challenge.includes("Script")
+challenge.split()
+challenge.split(" ")
+
+const companies = "Facebook, Google, Microsoft, Apple, IBM, Oracle, Amazon"
```
Comment (kRojecki) [inline 02_Day_Data_types/02_day_starter/main.js:41]: first You use let, now const for the same thing. Why?

File: 02_Day_Data_types/02_day_starter/main.js [hunk 4]
```diff
@@ -1 +1,150 @@
...
+challenge.charCodeAt(11)
+challenge.indexOf("a")
+challenge.lastIndexOf("a")
+
+const because = 'You cannot end a sentence with because because because is a conjunction'
+
+because.indexOf("because")
+because.lastIndexOf("because")
+because.search("because")
+' 30 Days Of JavaScript '.trim()
```
Comment (kRojecki) [inline 02_Day_Data_types/02_day_starter/main.js]: Why not assing that value "30 Days Of JavaScript" to variale?

Comment (kRojecki) [reply 02_Day_Data_types/02_day_starter/main.js]: oh wait, You have that variable -  challenge.

Comment (arojecka) [reply 02_Day_Data_types/02_day_starter/main.js]: changed

File: 02_Day_Data_types/02_day_starter/main.js (hunk 3 above)
Comment (arojecka) [reply 02_Day_Data_types/02_day_starter/main.js:41]: changed

File: 02_Day_Data_types/02_day_starter/main.js (hunk 2 above)
Comment (arojecka) [reply 02_Day_Data_types/02_day_starter/main.js]: fixed

File: 02_Day_Data_types/02_day_starter/main.js (hunk 1 above)
Comment (arojecka) [reply 02_Day_Data_types/02_day_starter/main.js]: fixed

**Comment by kRojecki** [comment]
excercise 1 looks fine.

**Pull request merged**